pulpcore/cli/common/context.py

```diff
@@ -60,6 +64,21 @@ def default(self, obj: Any) -> Any:
             return super().default(obj)
 
 
+def _preprocess_value(key: str, value: Any) -> Any:
+    if isinstance(value, PulpEntityContext):
+        return value.pulp_href
+    return value
+
+
+def preprocess_body(body: EntityDefinition) -> EntityDefinition:
```

Comment:
We now preprocess body _and_ parameters with this. So maybe it is not the best name for the function.

I'm going to call it `preprocess_payload`.